src/extrapolation.rs

```diff
@@ -83,7 +83,7 @@ use bevy::prelude::*;
 /// Then, add the [`TransformExtrapolationPlugin`] to the app with the velocity sources:
 ///
 /// ```
-/// use bevy::{ecs::query::QueryData, prelude::*};
+/// use bevy::{ecs::query::QueryData, time::TimePlugin, prelude::*};
```

Comment:
Those are slightly out of scope for this PR, but needed for bevy 0.15.1 due to https://github.com/Jondolf/bevy_transform_interpolation/pull/7.